### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by pariyar07**
Authentication added

### CODE REVIEW

File: src/components/playlistModal.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                <div className="playlist-modal-content" onClick={e => e.stopPropagation()}>
+                    <div className="playlist-modal-header">
+                        <h4>Create a Playlist</h4>
+                    </div>
+                    <div className="playlist-modal-body">
+                        <input type="text" placeholder="Add playlist name" onChange={(e) => setPlaylistName(e.target.value)} value={playlistName}/>
+                    </div>
+                    <div className="playlist-list">{playlist.map(play => {
+                        return <p key={uuid()}>{play.name}</p>
+                    })}</div>
```
Comment (darshandamre) [inline src/components/playlistModal.jsx]: this is not good, generating a uuid everytime for key would result in unnecessary re-renders, use playlist id

Comment (sanjitsarkar) [reply src/components/playlistModal.jsx]: Yes you can playlist._id

**Review by sanjitsarkar — APPROVED** [review]
Code looks clean and readable

File: src/components/playlistModal.jsx (hunk 1 above)
Comment (pariyar07) [reply src/components/playlistModal.jsx]: done

**Pull request merged**